plugins/woocommerce/src/Enums/OrderStatus.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Checkout\Enums;
```

Comment:
(1/2) Following https://github.com/woocommerce/woocommerce/pull/51919/commits/72ef37d4a0002ae764cb0ea1c97b870f3dc05b70, we should probably change this:

```suggestion
namespace Automattic\WooCommerce\Enums;
```